editor/scene/3d/node_3d_editor_plugin.cpp

```diff
@@ -4852,7 +4852,9 @@ Vector3 Node3DEditorViewport::_get_instance_position(const Point2 &p_pos, Node3D
 
 	HashSet<RID> rids;
 
-	if (!preview_node->is_inside_tree() && !ruler->is_inside_tree()) {
+	if (preview_node && preview_node->get_child_count() > 0) {
+		_insert_rid_recursive(preview_node, rids);
+	} else if (!preview_node->is_inside_tree() && !ruler->is_inside_tree()) {
```

Comment:
In this block there's an unused array, please remove it:
https://github.com/godotengine/godot/blob/ef34c3d534a16b140aef0643ed2fa36f8e9d1612/editor/scene/3d/node_3d_editor_plugin.cpp#L4860